packages/web/services/twitter.ts

```diff
@@ -0,0 +1,112 @@
+import { createClient } from "@vercel/kv";
+import axios from "axios";
+
+import { TWITTER_API_ACCESS_TOKEN, TWITTER_API_URL } from "~/config";
+
+const twitterApi = axios.create({
+  baseURL: TWITTER_API_URL,
+  headers: {
+    Authorization: `Bearer ${TWITTER_API_ACCESS_TOKEN}`,
+  },
+});
```

Comment:
I recommend using `fetch` as it's built-in and won't add to the bundle size. I've developed an [API client](https://github.com/osmosis-labs/osmosis-frontend/pull/2208/files#diff-ceb538631f37f488ab98b32f93c1cad7dd499df2fc8859f5657d2db817b4a886) in my work-in-progress PR #2208. Please consider incorporating it into this PR."

Reply:
Yes, I'll refactor that, I used axios because I had seen it around the codebase

Reply:
I saw that PR is still in draft, as soon as you implement it on stage branch, I will do a rebase and use your client, there are libraries built using only fetch that you might be interested in: https://github.com/sindresorhus/ky (maybe in the future for advanced cases)